holoviews/plotting/bokeh/callbacks.py

```diff
@@ -722,10 +722,9 @@ def _process_selection_event(self):
         # meaning the popup has already been removed; we need to regenerate
         if self._existing_popup and not self._existing_popup.visible:
             if position:
-                self._panel.position = XY(**position)
-                self._existing_popup.visible = True
                 if self.plot.comm:
                     push_on_root(self.plot.root.ref['id'])
+                self._panel.position = XY(**position)
```

Comment:
Why move this after the push_on_root? It won't be synced in a notebook.

Reply:
Woops, I forgot what it was for and I was trying to achieve some behavior so I randomly moved things around and since I used vscode, it didn't affect me. I reverted it.